### DISCUSSION TITLE
Set Custom Duration Times

### DISCUSSION THREAD

**Discussion by robs007** [gopro/labs#452] [Feature Requests]
I would like to be able to set custom duration recording times instead of just using the defaults,

**Comment by dnewman-gpsw** [comment]
You can do that with QR codes, setting start and stop times. e.g. !S!1000E  <- start now and end after 1000 seconds.   But the duration record feature has fixed lengths in the menu and Labs can't easily change those.  Is the workaround sufficient?

**Comment by robs007** [comment]
Would is start recording when I started it with the remote, or would I have to show it the QR code each time?
________________________________

**Reply by dnewman-gpsw** [reply]
This is show the QR Code.  You can script it to do this so that any capture, Remote, App or shutter button with be limited.

This script will limit all captures to 10s.
=B0<r0=At:B>r0=Bt:B+=B-A>B10!E!R

but you can change the length to anything (here is 957 seconds.)
=B0<r0=At:B>r0=Bt:B+=B-A>B**957**!E!R

**Comment by robs007** [comment]
That doesn't see m to work. I have everything set to off and put that code in the additional code box.

[cid:20cd7015-74c8-4561-b6bd-5b1079537c46]
________________________________

**Reply by dnewman-gpsw** [reply]
HERO11 only.  This stuff is pretty new.

**Comment by robs007** [comment]
Oh, ok gotcha. Thanks

On Jul 19, 2023, at 5:40 PM, David Newman ***@***.***> wrote:

﻿

HERO11 only. This stuff is pretty new.
